pkg/traceql/series_heap.go

```diff
@@ -1,17 +1,75 @@
 package traceql
 
+import "strings"
+
 // seriesValue keeps a value from a time series with its key
 type seriesValue struct {
 	key   SeriesMapKey
 	value float64
 }
 
+// compareSeriesMapKey compares two SeriesMapKey values deterministically.
+// Returns -1 if a < b, 0 if a == b, 1 if a > b
+func compareSeriesMapKey(a, b SeriesMapKey) int {
```

Comment:
nit: this code is mostly exercised in metrics tests but it would be nice to have a few tests for compareSeriesMapKey and compareSeriesValues

Reply:
done